content/blog/2024-07-02-tmix-06/index.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "This Month in Xilem, June 2024"
+authors = ["Daniel McNab"]
++++
+
+We released [Vello 0.2.0][] at the start of the month, which included some key improvements discussed in previous months' updates.
+In June, we merged initial handling of Emoji ([vello#615][]), which supports glyphs which use the COLR specification (so this does not, for example, support Apple's system Emoji).
+In-flight work includes an enhanced testing setup ([vello#610][]), GPU memory allocation robustness ([vello#606][]), and improved correctness for some unusual strokes ([vello#607][]).
+
+<figure>
+
+<img src="Xilem Emoji Picker.png" style="height: auto" alt="Screenshot of an app, containing a 4x4 grid of labelled Emoji, zoom in and out buttons and pagination controls. 'cat face with wry smile' is selected" height="1132" width = "863">
+
+<figcaption>
+
+An in-progress Emoji viewer example ([xilem#420][]) running on Android.
+This example currently only works for systems with COLR system emoji, such as some Android systems.
+
+</figcaption>
+</figure>
+
+[xilem#310][] brought in some new ideas about how Xilem Core can be structured, to move away from the macro heavy design used previously.
+I believe that this has additional advantages for specialised `View` types, such as OS windows, although this is yet to be integrated.
+Support for the `Adapt` views built on this has enabled an [Elm](https://elm-lang.org/)-inspired example for Xilem in [xilem#401][], showing the flexibility of the Xilem architecture.
+Our observability support has also been improved, with automatic tracing to a file in debug builds added in [xilem#384][]; [xilem#396][] will lay the foundation for more advanced debugging capabilities.
```

Comment:
xilem#396 isn't really about tracing or debugging. It's more of an internal architecture change, for making easier to write new features and slightly easier testing.